Branch-SDK/src/io/branch/referral/RemoteInterface.java

```diff
@@ -33,6 +33,7 @@
 
 	private static final String SDK_VERSION = "1.5.0";
 	private static final int DEFAULT_TIMEOUT = 3000;
+	private static int retryCount = 0;
```

Comment:
That's a relic of testing, good catch!

On Wednesday, April 15, 2015, Graham Mueller <notifications@github.com>
wrote:

> In Branch-SDK/src/io/branch/referral/RemoteInterface.java
> <https://github.com/BranchMetrics/Branch-Android-SDK/pull/61#discussion_r28473047>
> :
>
> > @@ -33,6 +33,7 @@
> >
> >  	private static final String SDK_VERSION = "1.5.0";
> >  	private static final int DEFAULT_TIMEOUT = 3000;
> > +	private static int retryCount = 0;
>
> Should this be here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BranchMetrics/Branch-Android-SDK/pull/61/files#r28473047>
> .
>


-- 

*Sahil Verma*

*Branch Metrics *

w branch.io | c 510-386-1750 | s sahilroyverma